NOnion/Crypto/HiddenServicesCipher.fs

```diff
@@ -12,6 +12,16 @@ open NOnion
 open NOnion.Utility
 
 module HiddenServicesCipher =
+    let SHA3256(bytes: array<byte>) =
```

Comment:
are you mixing a refactoring with a change of behaviour in this commit? if so, let's make the commit message title represent the behaviour change, and explain the refactoring part in the commit message body

Reply:
TBH these two changes should've been separated in the first place.

Reply:
Fixed